Html/login.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../Css/testStyle.css">
```

Comment:
It would be better to use lower case in directory naming
```suggestion
    <link rel="stylesheet" href="../css/testStyle.css">
```

Reply:
Мы изменили названия папок